apps/openchallenges/grafana/.env.example

```diff
@@ -0,0 +1,2 @@
+GF_SECURITY_ADMIN_USER=openchallenges
+GF_SECURITY_ADMIN_PASSWORD=changeme
```

Comment:
Do we need to add `SERVER_PORT=3000 ` for grafana, like we did for other services?

Reply:
Not for now because port 3000 is the default Grafana port and it does not conflict with other services. Ultimately I would like to control all the port, including the ones with default values. This is work for the future.